test/visibility/bug-15261/bug-15261-3.chpl

```diff
@@ -0,0 +1,32 @@
+module MHDF5 {
+  import SysCTypes;
+
+  public import this.C_HDF5;
+  module C_HDF5 {
+    import SysCTypes;
+
+    public import this.HDF5_WAR;
+    module HDF5_WAR {
+      proc H5LTmake_dataset_WAR() {
+        writeln("in H5LTmake_dataset_WAR");
+      }
+    }
+  }
+}
+
+module GenSymIO {
+  use MHDF5;
+
+  proc write1DDistArray() throws {
+    coforall i in 1..1 {
+      import C_HDF5.HDF5_WAR.HDF5_WAR.H5LTmake_dataset_WAR;
```

Comment:
Why is this intended to work?  You're listing the same module twice

Reply:
That was not intentional, thanks for noticing. Might be a reasonable issue though - whether or not that should work.

Reply:
I made issue #15265 about it.